cfpb/consumerfinance.gov
# Add missing Spanish translations to aria-labels

We have various Spanish language pages that have English `aria-label` attributes. This validates [accessibility guidance](https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA6.html) that

> the text description accurately labels the object or provides a description of its purpose or provides equivalent information

For Spanish language pages, having English `aria-label` attributes doesn't feel like it meets this requirement.

This PR adds missing translations for these attributes so that they render in Spanish on Spanish pages. See below for a detailed list of translations, and pages to use to test this PR.

Closes internal platform#3709.

## How to test this PR

Run a local production server, and view source on these pages to confirm the correct labels. Note that the various Ask CFPB links require a running Elasticsearch server populated with Ask CFPB content.

|English label|New Spanish translation|English page|Spanish page|
|-|-|-|-|
|Breadcrumbs|Guía de navegación|[link](http://localhost:8000/consumer-tools/financial-well-being/about/)|[link](http://localhost:8000/consumer-tools/financial-well-being/about/es/)|
|Next featured content item|Siguiente ítem destacado|[link](http://localhost:8000/)|[link](http://localhost:8000/es/)|
|Pagination|Número de página|[link](http://localhost:8000/ask-cfpb/search/?q=auto)|[link](http://localhost:8000/es/obtener-respuestas/buscar/?q=auto)|
|Play video|Ver video|[link](http://localhost:8000/coronavirus/mortgage-and-housing-assistance/director-message-housing-resources/)|[link](http://localhost:8000/es/coronavirus/asistencia-hipotecas-y-viviendas/)|
|Play video on YouTube|Ver video en YouTube|[link](http://localhost:8000/coronavirus/mortgage-and-housing-assistance/director-message-housing-resources/)|[link](http://localhost:8000/es/coronavirus/asistencia-hipotecas-y-viviendas/)|
|Previous featured content item|Ítem destacado previo|[link](http://localhost:8000/)|[link](http://localhost:8000/es/)|
|Search term|Palabra a buscar|[link](http://localhost:8000/consumer-tools/auto-loans/answers/)|[link](http://localhost:8000/es/herramientas-del-consumidor/prestamos-para-vehiculos/respuestas/)|
|Section navigation|Navegación de la sección|[link](http://localhost:8000/coronavirus/mortgage-and-housing-assistance/request-forbearance-or-mortgage-relief/)|[link](http://localhost:8000/es/coronavirus/asistencia-hipotecas-y-viviendas/solicite-aplazamiento-o-alivio-de-hipoteca/)|
|Main menu|Menú principal|[link](http://localhost:8000/)|[link](http://localhost:8000/es/)|

## Checklist
